File: demos/2.js-core/hello.js
var aaa = 1;
var zzz = "hello world";
zzz = "你好，世界";

function bbb () {
    alert("我是 bbb");
    return "hello";
}
// var bbb = function bbb () {
//     alert("我是 bbb");
// }
function fff (f) {
    f();

    var res = f();
    alert("我接受的这个函数" + res + "真的好");
    // alert("我接受的这个函数" + f() + "真的好");
}

fff(bbb);

// var ccc = bbb;
// var ddd = ccc;
// var eee = ddd;
// eee();

// alert(aaa);

File: demos/4.java-base/Hello.java
public class Hello {
    public static void main(String[] aaa) {
          System.out.println("我要打印参数了\n");
          for (int i = 0; i < aaa.length; i++) {
               System.out.println(i + " : " + aaa[i]);
          }
          System.out.println("打印结束. 总共打印了 " + aaa.length + " 个参数");
    }
}